src/test/java/org/opensearch/security/test/AbstractSecurityUnitTest.java

```diff
@@ -199,23 +199,7 @@ protected void initialize(ClusterInfo info, Settings initTransportClientSettings
             Assert.assertEquals(info.numNodes, cur.getNodes().size());
 
             SearchResponse sr = tc.search(new SearchRequest(".opendistro_security")).actionGet();
```

Comment:
Note; this is inline with main - all of these other requested slowed down how long startup took and these assertions aren't that useful since the plugin just throws exceptions if it couldn't create the security index.